spec/factories/users.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  first_name             :string
+#  last_name              :string
+#  opt_out                :boolean          default(FALSE)
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+FactoryBot.define do
+  factory(:user) do
+    email { Faker::Internet.email }
```

Comment:
To avoid repeats which will crash tests, try using `Faker::Internet.unique.email`.